lib/build-cron.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
```

Comment:
Renamed to avoid case sensitivity issues. Moved event handler code into new file.